pkg/tenant/tenants.go

```diff
@@ -67,54 +67,69 @@ func (s *Server) TenantList(c *gin.Context) {
 // Route: /tenant
 func (s *Server) TenantCreate(c *gin.Context) {
 	var (
-		err error
-		t   *api.Tenant
+		err    error
+		t      *tokens.Claims
+		tenant *api.Tenant
 	)
 
-	// TODO: Add authentication and authorization middleware
+	// Fetch tenant from the context.
+	if t, err = middleware.GetClaims(c); err != nil {
```

Comment:
I tend to name these variables `claims` or similar, since it refers to user claims and doesn't really have anything to do with a tenant (expect that it contains an orgid). That will probably help reduce confusion with the two other "tenant" variables in this method.